src/backend/app.js

```diff
@@ -21,10 +21,68 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+// nodejs week1
+// Respond with all meals in the future (relative to the when datetime)
+app.get("/future-meals", async (req, res) => {
+  try {
+    const row = await knex.raw("SELECT * FROM meal WHERE `when` > now()");
+    res.status(200).json(row[0]);
+  } catch (e) {
+    res.status(404).json(e);
+  }
+});
+
+//Respond with all meals in the past (relative to the when datetime)
+app.get("/past-meals", async (req, res) => {
+  try {
+    const row = await knex.raw("SELECT * FROM meal WHERE `when` < now()");
+    res.status(200).json(row[0]);
```

Comment:
better to add the following conditions:
if (row[0].length > 0) {
      res.status(200).json(row[0]);
    } else {
      res.status(404).json("there are no meals");
    }

Reply:
Thank you so much for the review. I will make the changes :heart_eyes:

Reply:
for this change its required in the homework to show empty string if there is no meals :man-gesturing-ok: